src/LazyHydrate.js

```diff
@@ -154,7 +154,7 @@ export default {
     },
   },
   mounted() {
-    if (this.$el.childElementCount === 0) {
+    if (this.$el.childElementCount === 0 || this.$el.nodeType !== 1 ) {
```

Comment:
Please use the node type constant `Node.ELEMENT_NODE` instead of `1` to make the code more straightforward. Thanks!